packages/wonder-blocks-button/types.js

```diff
@@ -0,0 +1,85 @@
+// @flow
+import React from "react";
+import type {Element} from "react";
+
+import type {ValidTints} from "wonder-blocks-color";
+
+export type SharedProps = {
```

Comment:
Even though it helps to cut down on duplicate code I think we may want to move this in to button.js (and parts of it into button-core.js) and merge it in with their existing props in order to ensure that the styleguidist prop documentation is generated correctly.